src/js/msp/MSPHelper.js

```diff
@@ -1692,6 +1692,13 @@ MspHelper.prototype.process_data = function(dataHandler) {
                 break;
             case MSPCodes.MSP_SET_FAILSAFE_CONFIG:
                 console.log('Failsafe config saved');
+                break;
+            case MSPCodes.MSP_OSD_CANVAS:
+                OSD.data.VIDEO_COLS['HD'] = data.readU8();
+                OSD.data.VIDEO_ROWS['HD'] = data.readU8();
+                OSD.data.VIDEO_BUFFER_CHARS['HD'] = OSD.data.VIDEO_COLS['HD'] * OSD.data.VIDEO_ROWS['HD'];
+                console.log(`Canvas ${OSD.data.VIDEO_COLS['HD']} x ${OSD.data.VIDEO_ROWS['HD']}`);
+
```

Comment:
Not important, but the rest of cases not include an empty line before the break, so better to maintain the coherence in the style.
This is a read MSP message? In theory all the "read" commands go higher, is easy to see them because they contain code, and we have here, at bottom only "SET" commands, that usually only write to the console.

Reply:
Blank line removed.